src/Objects/_PVector.js

```diff
@@ -0,0 +1,283 @@
+module.exports = function(options, undef) {
+  'use strict';
+
+  const TAU = 2 * Math.PI,
+        //TAU = options.PConstants.TAU,
+        lerp = function(start, stop, amt) {
+          return amt * (stop - start) + start;
+        },
+        sq = function(n) {
+          return n*n;
+        },
+        argsErr = function(mtd, len, min) {
```

Comment:
let's also include the function name so that if the throw is handled internally by Pjs, we can still see where things went wrong, even if the track trace is changed.